bundle/edu.gemini.dataman.app/src/main/scala/edu/gemini/dataman/query/JsonCodecs.scala

```diff
@@ -156,4 +156,47 @@ private[query] object JsonCodecs {
     )
 
   implicit val CodecJsonQaResponse = CodecJson.derived[QaResponse]
+
+  // ** EitherQaResponse
+
+  import GsaQaUpdateQuery.EitherQaResponse  // String \/ QaResponse
+
+  implicit val EncodeJsonEitherQaResponse: EncodeJson[EitherQaResponse] =
```

Comment:
This is probably a bit sketchy.  It works and the test cases pass but there is a predefined scalaz either codec provided by argonaut.  It might be worth it to define a specific simple type for the error case, define a codec for that which adds the quirky `result` field and then use the provided codec.  Perhaps.